config.json

```diff
@@ -1237,6 +1237,20 @@
       "unlocked_by": "hamming",
       "uuid": "ec5dab72-dd96-4dce-8886-aff52a6eb090"
     },
+    {
+      "core": false,
+      "difficulty": 4,
+      "slug": "rectangles",
+      "topics": [
+        "control-flow (conditionals)",
+        "control-flow (loops)",
+        "arrays",
+        "matrices",
+        "strings"
+      ],
+      "unlocked_by": "matrix",
```

Comment:
I would suggest the unlock for this be `tournament`. We have a few similar exercises there all around taking some ASCII representation of a thing and figuring that thing out.